pkg/gardenlet/controller/backupentry/backup_entry_reconciler.go

```diff
@@ -77,6 +78,15 @@ func (r *reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, err
 	}
 
+	// Remove the operation annotation if its value is not "restore"
```

Comment:
Nit: Maybe extend the doc string to explain why there is this special case for `restore`

Reply:
Done